download.html

```diff
@@ -439,361 +482,15 @@ <h2>Source Releases</h2>
                         <a href="https://archive.apache.org/dist/ignite/2.5.0/apache-ignite-2.5.0-src.zip.sha512">sha512</a>)
                     </td>
                 </tr>
-                <tr>
-                    <td style="white-space: nowrap;">2.4.0</td>
```

Comment:
Why do we remove previous releases from the list? It's ok to list all the previous versions. What we can do is to place all the old releases under a special section and make it folded by default (users need to click on "Previous Releses" to get the old versions).

Reply:
From my point of view 95% requests for download is to get only the latest release version. Why do we need to keep all Ignite's version on this page? Who will want to download Ignite 2.4 version?

I've kept this sentence under each table, so users will be able to download archive versions from the apache archive. 

```
<p>If you are looking for previous release versions of Apache Ignite, please have a look in the
                <a href="https://archive.apache.org/dist/ignite">archive</a>.
```

Reply:
Maxim, this sentence works. Agree, let's use it instead of the long list.